testing/test_cacheprovider.py

```diff
@@ -925,3 +925,15 @@ def test_does_not_create_boilerplate_in_existing_dirs(testdir):
     assert os.path.isdir("v")  # cache contents
     assert not os.path.exists(".gitignore")
     assert not os.path.exists("README.md")
+
+
+def test_cachedir_tag(testdir):
+    """Ensure we automatically create CACHEDIR.TAG file in the pytest_cache directory (#4278)."""
+    from _pytest.cacheprovider import Cache
+    from _pytest.cacheprovider import CACHEDIR_TAG_CONTENT
+
+    config = testdir.parseconfig()
+    cache = Cache.for_config(config)
+    cache.set("foo", "bar")
+    cachedir_tag_path = cache._cachedir.joinpath("CACHEDIR.TAG")
+    assert cachedir_tag_path.read_bytes() == CACHEDIR_TAG_CONTENT
```

Comment:
I think this could be just asserted in some existing test (to not increase number of tests unnecessarily).

Reply:
Perhaps mark the PR as "Changes requested" @blueyed? This makes clear that there are still issues to address here, both to @feuillemorte and by us (I almost merged this until I saw this comment).